tests/lib/processors/openrelik.py

```diff
@@ -42,8 +42,11 @@ def testPollWorkflowStatus(self, mock_download_workflow, mock_get_workflow):
     self._module.openrelik_workflow_client = workflows.WorkflowsAPI(
       api_client.APIClient("fake_api", "fake_key")
     )
-    status_generator = self._module.PollWorkflowStatus(456)
-    self.assertRaises(errors.DFTimewolfError, next, status_generator)
+    # Commenting this out since now the module won't raise a ciritical error
+    # Uncomment these two lines when critical errors are re-enabled.
+
+    # status_generator = self._module.PollWorkflowStatus(456)
```

Comment:
Rather than comment it out, what we want at the moment is to test that no error is raised.

Reply:
Done.